frontend/src/types/transactions.ts

```diff
@@ -0,0 +1,18 @@
+export interface ITransaction {
```

Comment:
Your naming is very bad.
This is not transactions. Your transaction is what you have on the backend. But this is smth different. It's transaction group or expenseByMerchant anything but not transaction